test/config.json

```diff
@@ -720,15 +720,6 @@
 			"RetryMax": 1,
 			"Tags": []
 		},
-		"vtgate_mysql80_derived": {
```

Comment:
The reason this became a dead link is that the test was moved from `derived` to `queries` and then the directory was deleted.
However, the test has not been added to config.json with the new path, which means that this test has actually been lost from CI. It could be broken and we won't know.
PR where original change happened: #11379 
I would have expected that this test failed on that PR before it was merged, but I haven't checked.
In any case, instead of deleting this, we should be renaming it to vtgate_queries_derived and fixing the path.

Reply:
That's a good catch! I'll fix that.

However, I will leave the tool as it is, it simply reports what are the dead links, then it is up to the user to delete or rename the cluster path.

Reply:
Agreed, No changes to the tool, just to config.json.
Once that is done, you may dismiss my review and merge this PR.